.github/workflows/push-build.yaml

```diff
@@ -3,6 +3,8 @@ on:
   push:
     branches:
       - master
```

Comment:
Adding the release branches to the list?

Reply:
we want to start the action when tags are pushed so I think having only tags should work.

Reply:
The behavior we need is:
1. If the master branch, publish and promote the build
2. If the release branch, publish the build
3. If a tag, publish and promote the build

We need to add the release branches to the list for the second bullet, right?

Reply:
okay, we have three events, when I was looking at the Jenkins file I saw only 1. ` for master` and `for release`, maybe add the tag on the running pipeline but let's discuss more.